source/gamefunctions_debug.bmx

```diff
@@ -1849,7 +1793,7 @@ Type TDebugAudienceInfos
 		DrawRect(0,0,800,385)
 		SetColor 255, 255, 255
 
-		'GetBitmapFontManager().baseFont.Draw("Bev�lkerung", 25, startY)
+		'GetBitmapFontManager().baseFont.Draw("Bev�lkerung", 25, startY)
```

Comment:
Schaut irgendwie ... komisch aus. (siehe auch Umlaute weiter unten).

Reply:
Da scheint was mit dem File-Encoding nicht zu passen. Ich bearbeite die Dateien in Eclipse und das Default-Encoding ist auf UTF-8 eingestellt. Die Datei muss also ursprünglich ein anderes Encoding gehabt haben. Welches Encoding ist denn für bmx-Dateien vorgesehen, damit die Strings korrekt angezeigt werden?
P.S.  bei mir sind in den Log-Ausgaben die Umlaute auch (oft) falsch.

Reply:
Also wenn man UTF8 drin hat, wird es UTF8 (ohne BOM). Ich benutze einfach nur Geany zum bearbeiten.

Die Konsolenausgabe -- weiss nicht warum dort kein UTF8 funzt - liegt glaube an der Windowskonsole (unter Linux sieht mein Text oft gut aus).